src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './Logo.styl';
+import logo from '../../img/logo.png';
```

Comment:
since you added: `resolve.extensions: ['.js', '.jsx', '.styl', '.gif', '.png', '.jpg', '.jpeg', '.svg']` you might avoid adding file extensions to your imports. 
Try to use them like this `import logo from '../../img/logo';` and see if it works

Reply:
done